### PR TITLE
Dependabot analyse GHA workflows

### PR DESCRIPTION
**Pull request by remcowesterhoud**
## Description

* Bump add-to-project to version v0.1.0
* Have dependabot create PR's for outdated actions

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+        versions:
+          - ">= 1.0.a, < 1.1"
+      - dependency-name: com.mycila:license-maven-plugin
+        versions:
+          - "4.0"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (korthout) [inline .github/dependabot.yml:22]: ❓ For the `maven` ecosystem the schedule is set to a specific timezone. I'm not sure what the benefit is, but should we do the same for the GHA ecosystems?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
-    versions:
-    - ">= 0.9_1.a, < 0.9_1.1000004"
-  - dependency-name: org.antipathy:mvn-scalafmt_2.12
-    versions:
-    - ">= 1.0.a, < 1.1"
-  - dependency-name: com.mycila:license-maven-plugin
-    versions:
-    - "4.0"
+  - package-ecosystem: maven
+    directory: "/"
```
Comment (korthout) [inline .github/dependabot.yml:4]: ❓ Any particular reason why the indentation changed? Was it not correctly formatted yaml? Is it currently formatted correctly?

Comment (remcowesterhoud) [reply .github/dependabot.yml:4]: Exactly, it wasn't properly formatted before.

File: .github/dependabot.yml (hunk 1 above)
Comment (remcowesterhoud) [reply .github/dependabot.yml:22]: It ha to do with the time dependabot start analysing. I'm not sure what the default start time is, but it really doesn't matter. It runs once a day which is fine. I don't care what time it starts.

**Pull request merged**